app/views/planning_applications/index.html.erb

```diff
@@ -9,10 +9,10 @@
       Your fast track applications
     </h1>
     <p class="govuk-body"><%= current_user.name %>, <%= current_user.role.capitalize %></p>
-    <% if exclude_others? && current_user.assessor? %>
+    <% if exclude_others? %>
       <p class="govuk-body"><%= link_to "View all applications", planning_applications_path, class: "govuk-link" %></p>
-    <% elsif !exclude_others? && current_user.assessor? %>
-      <p class="govuk-body"><%= link_to "View my applications", planning_applications_path(q: "exclude_others"), class: "govuk-link" %></p>
+    <% elsif !exclude_others? %>
```

Comment:
We could simplify this conditional slightly by just making it `else`

Reply:
Thank you! Fixed now.